### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by wojtekidd**
Updated Youtube video in Readme.md

### CODE REVIEW

**Comment by krancour** [comment]
Should be updated here as well: https://github.com/akuity/kargo/blob/3e008c94c3c3b49f59cde241314f63f55bd6d524/docs/docs/10-overview.md?plain=1#L24

**Comment by wojtekidd** [comment]
@krancour thanks for pointing that out - please check if everything is OK

File: docs/docs/10-overview.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Kargo's goal is to provide an intuitive and flexible layer "above" existing GitO
 Watch the *Multi-Stage Deployment Pipelines the GitOps Way* talk by Jesse Suen & Kent Rancourt of Akuity at GitOpsCon EU 2024.
 
 <center>
-<iframe width="560" height="315" src="https://www.youtube.com/embed/tl8w72tehZ0?si=fdSx63nsCQHR1eP0" title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture;" allowFullScreen></iframe>
+<iframe width="1280" height="720" src="https://www.youtube.com/embed/0B_JODxyK0w" title="Kargo - Multi-Stage Deployment Pipelines using GitOps - Jesse Suen / Kent Rancourt" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (krancour) [inline docs/docs/10-overview.md:24]: You've resized this to be too big even for the max column width.

Even the old width was too big on some smaller viewports.

Wondering if @rbreeze could give some guidance here to make this iframe constrained by the width of the containing element.

Comment (rbreeze) [inline docs/docs/10-overview.md:24]: yep! 
```suggestion
<iframe style="width: 100%" src="https://www.youtube.com/embed/0B_JODxyK0w" title="Kargo - Multi-Stage Deployment Pipelines using GitOps - Jesse Suen / Kent Rancourt" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

**Review by rbreeze — APPROVED** [review]

**Review by krancour — APPROVED** [review]

**Comment by krancour** [comment]
@rbreeze and I worked out the technical issues. Thanks @wojtekidd!

**Pull request merged**